geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/sortedset/AbstractZInterStoreIntegrationTest.java

```diff
@@ -78,6 +78,32 @@ public void shouldError_givenWrongKeyType() {
         STRING_KEY, KEY1)).hasMessage("WRONGTYPE " + RedisConstants.ERROR_WRONG_TYPE);
   }
 
+  @Test
+  public void shouldError_givenTwoSortedSetKeysAndWrongKeyType() {
```

Comment:
This test name might be a little clearer as something like "shouldReturnWrongTypeError_givenNonSortedSetKeyAsThirdSourceKey"